B/BiodiversityObservationNetworks/Package.toml

```
@@ -1,3 +1,3 @@
 name = "BiodiversityObservationNetworks"
 uuid = "a5b868d3-191d-4bba-a38a-ad28190da010"
-repo = "https://github.com/EcoJulia/BiodiversityObservationNetworks.jl.git"
+repo = "https://github.com/PoisotLab/BiodiversityObservationNetworks.jl"
```

Please keep the git extension, like all other packages in the registry:
```suggestion
repo = "https://github.com/PoisotLab/BiodiversityObservationNetworks.jl.git"
```